scripts/us_epa/national_emissions_inventory/process.py

```diff
@@ -0,0 +1,336 @@
+        """
+        df = df.replace({column_name: replace_metadata})
+        return df
+
+    def _regularize_columns(self, df: pd.DataFrame,
+                            file_path: str) -> pd.DataFrame:
+        """
+        Reads the file for national emissions data and regularizes the files into a 
+        single structure so that it can be processed at once. This includes dropping
+        additional columns, renaming the columns and adding the columns with null if
+        not present.
+
+        Args:
+            df (pd.DataFrame): provides the df as input
+            file_path (str): path to excel file as the input
+
+        Returns:
+            df (pd.DataFrame): provides the regularized df as output
+        """
+        if '2008' in file_path or '2011' in file_path:
+            df.rename(columns=replacement_08_11, inplace=True)
+            df['pollutant type(s)'] = 'nan'
+            if 'event' in file_path:
+                df['emissions type code'] = ''
+            elif 'process' in file_path:
+                df = df.dropna(subset=['fips code'])
+                df['emissions type code'] = ''
+            if '2008' in file_path:
+                df['year'] = '2008'
+            else:
+                df['year'] = '2011'
+        elif '2017' in file_path:
+            if 'Event' in file_path:
+                df['pollutant type(s)'] = 'nan'
+            elif 'point' in file_path:
+                if 'unknown' in file_path or '678910' in file_path:
+                    df.rename(columns=replacement_point_17, inplace=True)
+                df['emissions type code'] = ''
+            df['year'] = '2017'
+        elif 'tribes' in file_path:
+            df.rename(columns=replacement_tribes, inplace=True)
+            df = self._data_standardize(df, 'fips code')
+            df['pollutant type(s)'] = 'nan'
+            df['year'] = '2014'
+        else:
+            df.rename(columns=replacement_14, inplace=True)
+            if 'event' in file_path or 'process' in file_path:
+                df['emissions type code'] = ''
+            df['pollutant type(s)'] = 'nan'
+            df['year'] = '2014'
+        df = df[df_columns]
+        return df
+
+    def _national_emissions(self, file_path: str) -> pd.DataFrame:
+        """
+        Reads the file for national emissions data and cleans it for concatenation
+        in Final CSV.
+
+        Args:
+            file_path (str): path to excel file as the input
+
+        Returns:
+            df (pd.DataFrame): provides the cleaned df as output
+        """
+        df = pd.read_csv(file_path, header=0, low_memory=False)
+        df = self._regularize_columns(df, file_path)
+        df['pollutant code'] = df['pollutant code'].astype(str)
+        df['geo_Id'] = ([f'{x:05}' for x in df['fips code']])
+        #
+        # Remove if Tribal Details are needed
+        #
+        df['geo_Id'] = df['geo_Id'].astype(float).astype(int)
+        df = df.drop(df[df.geo_Id > 80000].index)
```

Comment:
define 80000 as a variable at the top of the file

Reply:
Done the changes.